google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +184,18 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def update_schema():
+        """Maintain all the schema changes for this table.
+
+        Returns:
+            dict: A mapping of Action: Column.
+        """
+        columnMapping = {'CREATE': Column('resource_name',
```

Comment:
What if we want to add more columns in the future?  Shouldn't we use a list?:
```
columnMapping = {'CREATE', [column1, column2, ..]}
```

Reply:
Updated.